src/auth-service/routes/v1/users.js

```diff
@@ -108,7 +108,7 @@ router.post(
 );
 
 router.post(
-  "/emailAuth/:purpose",
+  "/emailAuth/:purpose?",
```

Comment:
Hi @BenjaminSsempala , what is the purpose of the "?".

Thanks!

Reply:
Hey @Baalmart .
This makes the param optional.
The reason why it was giving "endpoint doesn't exist" was because it was expecting a compulsory parameter.

Reply:
Okay @BenjaminSsempala